dbt_project.yml

```diff
@@ -1112,3 +1112,8 @@ models:
       +schema: aerodrome
       base:
         +schema: aerodrome_base
+
+    hivemapper:
+      +schema: hivemapper
+      solana:
+        +schema: hivemapper_solana
```

Comment:
you can remove modifications on this file, we're working towards cleaning out schema assignments here and prefer in the model directly (which you already have)